evap/student/tests/test_views.py

```diff
@@ -706,3 +706,17 @@ def test_choice_fields_correctly_populated_by_server(self):
 
         self.assertEqual(form[normal_question_id].value, "-1")
         self.assertIsNone(form[dropout_question_id].value)
+
+    def test_change_language_in_dropout(self):
+        url_dropout = reverse("student:drop", args=[self.evaluation.id])
+        url_vote = reverse("student:vote", args=[self.evaluation.id])
+        page = self.app.get(url=url_dropout, user=self.user, status=200)
+        html_site = page.body.decode()
+
+        # do not redirect to vote page
+        self.assertNotIn(f"{url_vote}?language=en", html_site)
+        self.assertNotIn(f"{url_vote}?language=de", html_site)
```

Comment:
Would this assertion also pass? If yes, I'd prefer to use that (so that this assertion does not _just_ pass because the query part of the URL changed)

```suggestion
        # do not redirect to vote page
        self.assertNotIn(url_vote, html_site)
```

If this doesn't work, disregard